src/ui/render.ts

```diff
@@ -184,6 +189,46 @@ export default class UI {
     }
   }
 
+  createContextMenu() {
+    // Create context menu container
+    this.tableContextMenuDom = document.createElement("div");
+    this.tableContextMenuDom.id = "contextMenu";
+    this.tableContextMenuDom.classList.add("lightsheet_table_context_menu");
+
+    // Create options for the context menu
+    // TODO: Add working options and remove placeholders Copy, Cut, Paste
+    const options = ["Copy", "Cut", "Paste"];
```

Comment:
Could this be defined in `LightsheetOptions` along with method hooks for when the option is selected? That way we could allow extending with minimal effort.

Reply:
This is done now.